source/android/adaptivecards/CMakeLists.txt

```diff
@@ -75,7 +75,6 @@ add_library( # Sets the name of the library.
              ../../shared/cpp/ObjectModel/ToggleVisibilityAction.cpp
              ../../shared/cpp/ObjectModel/ToggleVisibilityTarget.cpp
              ../../shared/cpp/ObjectModel/UnknownElement.cpp
-             ../../shared/cpp/ObjectModel/UnknownAction.cpp
```

Comment:
is this by design ? This may be a bad merge conflict change?

Reply:
I never made this change.